requirements/prod

```diff
@@ -1,3 +1,3 @@
-jsonpickle
-tornado
-click
+jsonpickle==0.9.2
+tornado==4.2.1
+arteria==0.1.0
```

Comment:
Do we want to use github direct dependencies here? Like: `git+https://github.com/arteria-project/arteria-core.git` possibly also with a tag for the version? Until we get it into a package manager that should make it easier to build. Right now travis is failing the build for this reason.